docs/handbook/common-tasks.md

```diff
@@ -65,15 +65,47 @@ ITEMS TO WRITE ABOUT:
 </div>
 
 
-## Multimedia
+## Multimedia - Audio
 
-< place holder >
+<i class="fa fa-info-circle fa-lg" aria-hidden="true"></i> **DOC TEAM NOTE:**
+<div class="well">
+ITEMS TO WRITE ABOUT:
+
+* How to use the hidden `gstreamer-properties` configuration utility.
+* Pulse Audio configuration
+* [possible resource](https://www.solarismultimedia.com/)
+* [How it works](http://tuxradar.com/content/how-it-works-linux-audio-explained)
+
+</div>
+
+OpenIndiana employs several different audio components which work together to facilitate audio output and control.
+These systems are, in a sense, layered and each provides specific functions, although not without a significant a degree of crossover.
+The reason for so many different systems is largely a case of ensuring support for legacy as well as modern applications.
+
+| System | Description
+| --- | ---
+| Gstreamer | Mixer/Proxy
+| PulseAudio | Mixer/Proxy
+| OSS/OSSv4 | Open Sound System
+| Sun Audio | Solaris Audio API (Dev Audio)
```

Comment:
OK, sounds reasonable. I'll run it by Garret and also refine it based on what you have just written.

I'm also not sure what level of detail we should provide. My thoughts are providing a brief introduction as well as listing and or linking to man pages. Some information about how to adjust and select things, along with some guidance for testing should be sufficient I would think.